docs/installation.rst

```diff
@@ -33,18 +35,21 @@ pysat itself may be installed from a terminal command line via::
    pip install pysat
 
 There are a few packages that pysat depends on that will be installed as
-needed by the installer
+needed by the installer:
 
-     * dask
-     * netCDF4
-     * numpy
-     * pandas
-     * portalocker
-     * scipy
-     * toolz
-     * xarray
+#. dask
+#. netCDF4
+#. numpy
+#. pandas
+#. portalocker
+#. pytest
```

Comment:
pytest is only required for development, no?

Reply:
Hmm... the test classes are imported along with pysat for the downstream packages, so I think this might be a required package.  It's listed in both the requirements file and in the setup metadata as such.